.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -25,11 +25,18 @@ body:
         - Queues
         - R2
         - Workers for Platforms
-        - Workers/Other
-        - Wrangler
+        - Constellation
+        - Wrangler core
+        - Other
   - type: input
     attributes:
-      label: What version of `Wrangler` are you using?
+      label: What version of Wrangler are you using?
+      placeholder: 0.0.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: What version of Node are you using?
       placeholder: 0.0.0
     validations:
       required: true
```

Comment:
Do we need the node.js version to be required?

Reply:
changed to non-required